test_files/global_bug.js

```diff
@@ -0,0 +1 @@
+console.log([] instanceof global.Array);
```

Comment:
Lovely test! Can we also add a test for things that are completely missing from the context's global. Here's a list, make your pick.

```
process
URL
URLSearchParams
clearImmediate
clearInterval
clearTimeout
setImmediate
setInterval
setTimeout
Buffer
```

Reply:
```suggestion
console.log([] instanceof global.Array);
console.log(Boolean(process));
```